client-app/shared/layout/composables/useNavigations.ts

```diff
@@ -17,7 +15,7 @@ const mainMenuLinks = computed<MenuLink[]>(() =>
       ({
         id: item.id,
         route: item.route,
-        title: globals.i18n!.global.t(item.title),
+        title: globals.i18n.global.t(item.title),
```

Comment:
I think this should be a bit different, it's not clear how to use this. Maybe we should have an ability to use `useI18n` here, or use i18n globals from App instance, or pass data as-is and localize it within the template later.